src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 15px 5px 42px;
+  }
+
+  &__name {
+    margin: 0;
+    margin-bottom: 5px;
+
+    line-height: 15px;
+    font-size: 12px;
+    font-weight: 500;
+  }
+
+  &__code {
+    line-height: 13px;
+    font-size: 11px;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 15px;
+    margin-bottom: 20px;
+  }
+
+  &__stars--4 {
+    display: flex;
+  }
+
+  .stars__star {
+    background: url(../images/star.svg) no-repeat center;
+    width: 15px;
+    height: 15px;
+    margin-right: 4px;
+  }
+
+  .stars__star:last-child {
+    margin-right: 0;
+  }
+
+  &__stars--1 .stars__star:nth-child(-n+1),
+  &__stars--2 .stars__star:nth-child(-n+2),
+  &__stars--3 .stars__star:nth-child(-n+3),
+  &__stars--4 .stars__star:nth-child(-n+4),
+  &__stars--5 .stars__star:nth-child(-n+5) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__reviews {
+    display: inline-block;
+    line-height: 13px;
+    font-size: 11px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+
+    &__word {
+      line-height: 13px;
+      font-size: 11px;
+    }
+  
+    &__value {
+      line-height: 19px;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    display: block;
+    width: 170px;
+    height: 40px;
+    margin-top: 20px;
+
+    border: 1px solid #00acdc;
```

Comment:
use variables for the main values so that you'll be able to reuse them and give them descriptive names. But don't overuse them, don't create variable for the value that's used just once.